packages/generator-single-spa/src/html/templates/src/orgName-projectName.js

```diff
@@ -0,0 +1,14 @@
+import singleSpaHtml from "single-spa-html";
+import template from "./template.html";
+import "./styles.css";
+
+const htmlLifecycles = singleSpaHtml({
+  template,
+});
+
+export const { mount, unmount } = htmlLifecycles;
+
+/* Can also execute JavaScript after application has been mounted
+export const mount = (props) => htmlLifecycles.mount(props)
+  .then(() => { console.log("js after mount"); });
```

Comment:
Is it worth mentioning this?

Reply:
Since I don't know anyone who actually uses single-spa-html, it's hard to say.